albatross.ml

```diff
@@ -363,7 +403,7 @@ struct
         (* now we tell albatross about it, using a command for throwing it away *)
         (* note that the 'certs' / 'gen_cert' uses the policies for intermediate certificates *)
         query t ~domain (`Unikernel_cmd `Unikernel_info) >|= function
-        | Ok _ -> Ok ()
+        | Ok _ -> Ok (Vmm_trie.collect name t.policies)
```

Comment:
I struggle with this: could you explain what the plan is with this return value? And why it is a "collect name t.policies"? I'd expect that maybe the policy passed in here (which gets installed to albatross) will be returned in the success case.

Reply:
I just refactored it.

Reply:
After setting the policy, a good return could just be the policy itself. and we can pass that on as an api response